### DISCUSSION TITLE
How to run boofuzz with 0.0.0.0 host IP ?

### DISCUSSION THREAD

**Discussion by zhouqianyi** [jtpereyda/boofuzz#616] [Q&A] [answered]
When I run boofuzz, the UI page can be accessed through  http://localhost:26000  or  http://127.0.0.1:26000,but not  http://local ip:26000. I think the IP monitored by port 26000 is 127.0.0.1 instead of 0.0.0.0, How can I run with 0.0.0.0 host IP.

**Comment by cq674350529** [comment, marked as answer]
Hi, there is a recent change related to this behaviour. Specifically, another option `web_address` is added to `Session` class, which is `localhost` in default.
https://github.com/jtpereyda/boofuzz/blob/9080c4353bf4ce5a84b74a5752e3e07bd3243969/boofuzz/sessions.py#L458-L464

If you want to run with `0.0.0.0`, maybe you can try to set `web_address='0.0.0.0'` when initializing a `Session`.

**Reply by jtpereyda** [reply]
Thanks @cq674350529 ! This is a security-focused change, as listening on an external IP can be dangerous.
